examples/examples_with_aws/Prompt_Engineering_with_Llama_2_On_Amazon_Bedrock.ipynb

```diff
@@ -0,0 +1,2079 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prompt Engineering with Llama 2 - Using Amazon Bedrock + LangChain\n",
+    "\n",
+    "Prompt engineering is using natural language to produce a desired response from a large language model (LLM).\n",
+    "\n",
+    "This interactive guide covers prompt engineering & best practices with Llama 2.\n",
+    "\n",
+    "### Requirements\n",
+    "\n",
+    "* You must have an AWS Account\n",
+    "* You have access to the Amazon Bedrock Service\n",
+    "* For authentication, you have configured your AWS Credentials - https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html\n",
+    "\n",
+    "### Note about LangChain \n",
+    "The Bedrock classes provided by LangChain create a Bedrock boto3 client by default. Your AWS credentials will be automatically looked up in your system's `~/.aws/` directory\n",
+    "\n",
+    "#### Example `/.aws/config`\n",
+    "    [default]\n",
+    "    aws_access_key_id=YourIDToken\n",
+    "    aws_secret_access_key=YourSecretToken\n",
+    "    aws_session_token=YourSessionToken\n",
+    "    region = [us-east-1]\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Why now?\n",
+    "\n",
+    "[Vaswani et al. (2017)](https://arxiv.org/abs/1706.03762) introduced the world to transformer neural networks (originally for machine translation). Transformers ushered an era of generative AI with diffusion models for image creation and large language models (`LLMs`) as **programmable deep learning networks**.\n",
+    "\n",
+    "Programming foundational LLMs is done with natural language – it doesn't require training/tuning like ML models of the past. This has opened the door to a massive amount of innovation and a paradigm shift in how technology can be deployed. The science/art of using natural language to program language models to accomplish a task is referred to as **Prompt Engineering**."
```

Comment:
This is the original text from the Notebook here - https://github.com/facebookresearch/llama-recipes/blob/main/examples/Prompt_Engineering_with_Llama_2.ipynb. 

I do agree that it can create confusion, so will consider a subsequent PR with the update for both notebooks for clarity.